src/annotator/components/AdderToolbar.tsx

```diff
@@ -93,11 +91,22 @@ function ToolbarButton({
       )}
       onClick={onClick}
       title={title}
+      size="custom"
+      variant="custom"
     >
-      {Icon && <Icon className="text-annotator-lg" title={title} />}
-      {typeof badgeCount === 'number' && <NumberIcon badgeCount={badgeCount} />}
-      <span>{label}</span>
-    </ButtonBase>
+      <div
+        className={classnames(
+          'flex flex-col items-center gap-y-1 py-2.5 px-2',
+          'text-annotator-sm leading-none'
+        )}
+      >
```

Comment:
Where possible, I'm pulling layout styles or non-button-related styles off of `Button` and applying them to other elements. In this case an additional container `div` provided a way to apply local styling without pushing `classes` to `Button`.